src/polaris-log-properties.ts

```diff
@@ -6,12 +6,13 @@ export interface PolarisLogProperties {
     logId?: string;
     customProperties?: any;
     response?: any;
-    messageId?: string;
+    recordId?: string;
     eventKind?: string;
     eventKindDescription?: EventKindDescription;
     reality?: Reality;
     request?: Request;
     entity?: Entity;
+    entities?: Entity[];
```

Comment:
consider maybe to create a property named "entityOrEntities" which will be of type Entity | Entity[] and combine the 2 properties into 1, and by that to prevent from the clients the option to set the 2 properties in the same log. you will need also to change the "setEntityOrEntities" method.